pyads/ads.py

```diff
@@ -80,13 +83,12 @@
 )
 
 from .structs import (
-    AdsSymbol,
     AmsAddr,
     SAmsNetId,
     AdsVersion,
     NotificationAttrib,
     SAdsNotificationHeader,
-    SAdsSumRequest,
+    SAdsSumRequest
```

Comment:
Please don't kill the comma at the end of the line within brackets. It makes version control much easier with them.